scIB/integration.py

```diff
@@ -324,22 +324,37 @@ def runConos(adata, batch, hvg=None):
     return out
     """
     
+def runCombat(adata, batch):
+    sc.pp.combat(adata, key=batch)
+    return adata
 
 
-if __name__=="__main__":
-    adata = sc.read('testing.h5ad')
-    #emb, corrected = runScanorama(adata, 'method', False)
-    #print(emb)
-    #print(corrected)
+def runDESC(adata, batch, res=0.8, ncores=24, tmp_dir='/localscratch/'):
```

Comment:
Is it necessary to set 24 cores manually?

Reply:
It will otherwise use half the ones available... So I think we should hardcode the full amount we have, and then let it use half available when there are less than that on the system. Does that make sense to you?

Reply:
ok, sounds fair. We could also do `os.cpu_count()`

Reply:
that does sound better! Let's hope that works on the server as well... would need to test again I guess.

Reply:
c2eb7e2